app/assets/javascripts/components/students/students_filter.es6.jsx

```diff
@@ -17,21 +17,28 @@ class StudentsFilter extends Component {
   }
 
   get styles() {
-    return {
-      container: {
-        display: 'flex',
-        flexFlow: 'column',
-        justifyContent: 'center',
-        flex: '1',
-        height: '22px',
-        overflow: 'hidden',
-        zIndex: '100',
-      },
-      expanded: {
-        height: '100px',
-        overflow: 'scroll',
-      },
-    };
+      return {
+        container:
+          Object.assign(
+            {},
+            {
+              display: 'flex',
+              flexFlow: 'column',
+              justifyContent: 'center',
+              flex: '1',
+              height: '22px',
+              paddingLeft: '4px',
+              overflow: 'hidden',
+              zIndex: StyleConstants.zIndex.seven,
+            }
+          ),
+        expanded:
+          {
```

Comment:
Brace should just follow `expanded: {`. Check out some of the other files for reference!